test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -400,7 +400,7 @@ public async Task GetTrustResultAsync_WithUnavailableRevocationInformationInRequ
             SigningTestUtility.AssertRevocationStatusUnknown(matchingIssues, LogLevel.Warning, NuGetLogCode.NU3018);
         }
 
-        [PlatformFact(Platform.Windows, Platform.Linux, Skip = "https://github.com/NuGet/Client.Engineering/issues/1484")] // https://github.com/NuGet/Home/issues/11178
+        [PlatformFact(Platform.Windows, Platform.Linux, Skip = "https://github.com/NuGet/Home/issues/12186")] // https://github.com/NuGet/Home/issues/11178
```

Comment:
Why not using the original issue?

```suggestion
        [PlatformFact(Platform.Windows, Platform.Linux, Skip = "https://github.com/NuGet/Home/issues/11178")]
```

Reply:
I think https://github.com/NuGet/Home/issues/11178 is created when skipping the tests on Mac, and https://github.com/NuGet/Home/issues/12186 is created when skipping the tests on Linux and Windows.